hack/vagrant/script/install_kubernetes.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBERNETES_VERSION="v1.10.2"
+CRI_VERSION="v1alpha2"
+CNI_VERSION="v0.6.0"
+MASTER_CIDR="10.244.0.0/16"
+
+apt-get install -y ebtables socat
+
+# Configuring pouch's service to enable cri for kubelet
+mkdir -p /etc/systemd/system/pouch.service.d
+cat > /etc/systemd/system/pouch.service.d/pouch.conf <<EOF
+[Service]
+ExecStart=
+ExecStart=/usr/bin/pouchd --enable-cri=true --cri-version=$CRI_VERSION
+EOF
+systemctl daemon-reload
+systemctl restart pouch
+
+# Installing kubernetes's components
+wget -O- --read-timeout=20 --progress=bar:force:noscroll https://raw.githubusercontent.com/manlge/resources/master/kubernetes/$KUBERNETES_VERSION/kubernetes-linux-amd64.tar.xz|tar -C /usr/bin -xJ
```

Comment:
could we use the function to wrap the steps? it can make the shell script more readable and no global var pollution？ how do you think?